src/ui/EditorNS/editor.cpp

```diff
@@ -656,8 +646,8 @@ namespace EditorNS
             return theme;
 
         auto editorPath = QFileInfo(Notepadqq::editorPath());
-        auto bundledThemesDir = QDir(editorPath.absolutePath() + "/libs/codemirror/theme/");
-        auto themeFile = bundledThemesDir.filePath(name + ".css");
+        auto bundledThemesDir = QDir(editorPath.absolutePath() % "/libs/codemirror/theme/");
+        auto themeFile = bundledThemesDir.filePath(name % ".css");
```

Comment:
I don't think using `%` will give any improvements here. `QStringBuilder` delays construction of the `QString` until all concatenations a processed. But with just one concatenation there's nothing to delay.